content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -12,6 +12,7 @@ keywords:
   - trust
   - single-mesh
   - multiple-meshes
+  - mesh-expansion
```

Comment:
```suggestion
  - virtual-machines
```

Reply:
I don't think "mesh expansion" is the right phrase ... nobody knows what that means initially :). We should say VMs, or something like that.